### ISSUE TITLE
Link bus tooltips to line profile page

### ISSUE THREAD

**Issue by NoamGaash** [hasadna/open-bus-map-search#457]
In every map, when clicking a bus icon, there is a tooltip with information regarding the bus.

Add a button that will link to `/profile/routeId`

![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/570797a3-3da1-4c74-b5ce-e84c0c2013b0)

**Comment by YuvalMasada** [comment]
ok,
I've made a Small change in The BusToolTip component. I've added the following code, which doesn't change the old behavior.
`const LinkToProfile = siriRide ? (
    <Link to={`/profile/${siriRide.gtfsRouteRouteShortName}`}>
      {siriRide && siriRide.gtfsRouteRouteShortName}{" "}
    </Link>
  ) : null;`
Adding a screenshot of the new look:
![ToolTip look](https://github.com/hasadna/open-bus-map-search/assets/5049780/e553e4db-fdf6-481a-b21a-9b9c907ecc7f)

if it ok, then i commit it.
